tests/unit/db/views_tests.py

```diff
@@ -265,5 +284,144 @@ def test_custom_result_context_manager(self):
                 i += 1
             self.assertEqual(i, 100)
 
+class QueryIndexViewTests(unittest.TestCase):
+    """
+    QueryIndexView class unit tests.  These tests use a mocked DesignDocument
+    since a QueryIndexView object is not callable so an actual connection
+    is not necessary.
+    """
+
+    def setUp(self):
+        """
+        Set up test attributes
+        """
+        self.ddoc = mock.Mock()
+        self.ddoc.r_session = 'mocked-session'
+        self.ddoc.document_url = 'http://mocked.url.com/my_db/_design/ddoc001'
```

Comment:
Changed to use example.com.  Done in https://github.com/cloudant/python-cloudant/commit/cb9828a55aaa18a1e9b05db4eed94b94dcabaa3b.